github/lib/configure_repos.rb

```diff
@@ -34,14 +35,29 @@ def remove_old_webhooks!
       end
   end
 
+  def verify_repo_tags!
+    # TODO: make these work
+    govuk_repo_names = JSON.load(URI.open("https://docs.publishing.service.gov.uk/repos.json")).map { |repo| repo["app_name"] }
+    github_repo_names = github_repos_tagged_govuk.map { |repo| repo["name"] }
+    
+    untagged_govuk_repo_names = govuk_repo_names - github_repo_names
+    falsely_tagged_govuk_repo_names = github_repo_names - govuk_repo_names
+    # TODO: print these out
```

Comment:
Are these TODO comments accurate? I.e. are they still things to do, or should they be removed?

Reply:
Apologies, I have overlooked them, they've now been removed, thank you for highlighting them!